README.txt
// $Id: README.txt,v 1.2.2.8 2009/12/17 19:45:50 hswong3i Exp $

ABOUT INTERACTIVE MEDIA
----------------------

This is a re-implementation of the Interactive Media theme by Luka
Cvrk. This is a tableless, multi-column, fluid width layout. Parts
of the design are ported from Internet Services and Contented7 theme.

Official Interactive Media project page:
  http://www.solucija.com/templates/demo/Interactive_Media/

Drupal Interactive Media project page:
  http://drupal.org/project/interactive_media/
  http://pantarei-design.com/projects/interactive_media/

Sponsored by:
  http://pantarei-design.com/


HOWTO CUSTOMIZE THIS THEME?
----------------------

For theme customization usually you will need to change the style.css.
By the way this traditional method is not too flexible.

This theme provide a hook called as custom.css. The custom.css will always
override default style.css setting, and will not be covered during version
upgrade. Benefit of using this hook including:

  1. Your customization will NOT get mixed with original theme style.
  2. Your customization will NOT be override during theme upgrade.

This theme also provide some example for customization within
custom.example.css, divided into section by section. You can copy-and-paste
them to your custom.css and feel free to change it.


HOW THE MISSION HACK FUNCTION?
----------------------

By default, mission statement is protected by filter_xss_admin() so no
PHP script is allowed. On the other hand, all tags that can be used inside
an HTML body is allowed.

This theme provide a fancy hack so PHP script will able to execute if
mission statement starting with "<?php", or else will preform as default
style.

One example usage is combine with adsense module so only display a
skyscraper (120x600) at top page; another example is combine with banner
module so you will have a benner rotation as mission statement; or even
just execute a simple PHP code snippet as welcome message when user login.


ABOUT RTL SUPPORT (>=6.x)
----------------------

This theme is RTL supported, and fully tested with Acid2 compatible
browsers, e.g. FireFox3, Opera9.2 and Safari3. However, other browser
such as FireFox2 and Internet Explorer 6/7 may looks buggy.

As the implementation is validate with XHTML and CSS2 coding standard,
I am not going to provide browser-specific hack, for both LTR and RTL.

For more information about Acid2:
  http://en.wikipedia.org/wiki/Acid2

To test your browser with Acid2:
  http://www.webstandards.org/files/acid2/test.html


LIST OF MAINTAINERS
----------------------

PROJECT OWNER
M: Edison Wong <hswong3i@gmail.com>
S: maintained
W: http://edin.no-ip.com/


CHANGELOG.txt
// $Id: CHANGELOG.txt,v 1.1.1.1.4.69 2009/12/17 19:47:00 hswong3i Exp $

Interactive Media 6.x-1.12, 2009-12-18
----------------------
- update README.txt.

Interactive Media 6.x-1.11, 2009-05-14
----------------------
- better visibility for visited links.
- new custom style for print.module.
- header logo's div compatibility hack for IE8.
- [#344551]: fix primary links disconnected within small screen size.
- revamp primary links handling, clone from garland.
- [#359498]: document example about howto change the sidebar width.
- document more information about custom.css and custom.example.css.
- simplify custom theme function implementation, clone from garland.

Interactive Media 6.x-1.10, 2008-11-23
----------------------
- add @ingroup for custom.example.css, better documentation.
- update README.txt.
- better comment message.
- [#314124]: fix profile picture alignment.
- [#337338]: use '$front_page' instead of $base_path in page.tpl.php

Interactive Media 6.x-1.9, 2008-08-17
----------------------
- revamp primary links, better style and comment.
- add more clipart, sync with MediaWiki theme development.
- better h1/h2/etc styling for page layout arragement.
- [#262772]: fix truncated sitename in IE6/7.
- renew screenshot-drupal.org.png.

Interactive Media 6.x-1.8, 2008-06-19
----------------------
- simplify pencil link implementation.

Interactive Media 6.x-1.7, 2008-06-16
----------------------
- minor bugfix.
- CSS syntax cleanup.
- [#268850]: fix pencil link for Clear URL.
- add .meta-header and .meta-footer, with different styling.
- update large screenshot.
- update custom.example.css.

Interactive Media 6.x-1.6, 2008-05-26
----------------------
- add styling for tagadelic.module.
- add admin style, clone from garland.
- update large screenshot.
- reposition code layout, clone from garland.
- revamp page region handling, clone from garland.

Interactive Media 6.x-1.5, 2008-05-10
----------------------
- add a lot of fancy icons for meta data.
- update screenshots.
- better meta date style with height and line-height.

Interactive Media 6.x-1.4, 2008-05-07
----------------------
- add placeholders for IE6 fix.
- update document.
- update screenshots.
- bugfix fluid width layout, if override as fixed width in custom.css.
- bugfix fixed width example in custom.example.css.
- bugfix fluid width layout for IE6.
- bugfix background images for IE6.
- bugfix height and line-height for breadcrumb and block title.

Interactive Media 6.x-1.3, 2008-05-01
----------------------
- different hx.title sizing.

Interactive Media 6.x-1.2, 2008-04-28
----------------------
- better small screen layout.
- bugfix fluid width latout.

Interactive Media 6.x-1.1, 2008-04-27
----------------------
- bugfix fluid width layout in case of small screen.
- bugfix breadcrumb region as always preserve area.
- update custom.example.css.

Interactive Media 6.x-1.0, 2008-04-26
----------------------
- better admin/build/block style.
- fix footer region location.
- tricky hack: don't display header-region if empty, fix buggy IE7 display.

Interactive Media 6.x-1.0-rc3, 2008-04-25
----------------------
- bugfix content title styling if no sidebar.
- better regional handling, clone from garland.
- revamp background images.
- better LTR and RTL CSS coding.

Interactive Media 6.x-1.0-rc2, 2008-04-24
----------------------
- add experimental RTL support. tested with FF3, Opera9.2 and IE7 (thanks for Zohar).
- coding style cleanup.

Interactive Media 6.x-1.0-rc1, 2008-04-24
----------------------
- initial RC release.
- re-implementation of http://www.solucija.com/templates/demo/Interactive_Media/
- style port from http://drupal.org/project/contented7/ and http://drupal.org/project/internet_services/
- tableless, multi-column, fluid width layout.
- can customize layout with custom.css.
- CHANGE: change a and a:hover style.
- CHANGE: add logo, breadcrumb, ul/li style, meta style, comment/node/sticky style.
- TODO: better header region.
- TODO: RTL support.
- TODO: color.module support.

Interactive Media 5.x-1.11, 2009-12-18
----------------------
- update README.txt.

Interactive Media 5.x-1.10, 2008-11-23
----------------------
- sync with 6.x-1.x-dev.
- this should be the last stable release for Drupal 5.x.

Interactive Media 5.x-1.9, 2008-08-17
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.8, 2008-06-19
----------------------
- clone D6 $body_classes, so share style.css with D6.
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.7, 2008-06-16
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.6, 2008-05-26
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.5, 2008-05-10
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.4, 2008-05-07
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.3, 2008-05-01
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.2, 2008-04-28
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.1, 2008-04-27
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.0, 2008-04-26
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.0-rc3, 2008-04-25
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.0-rc2, 2008-04-24
----------------------
- sync with 6.x-1.x-dev.

Interactive Media 5.x-1.0-rc1, 2008-04-24
----------------------
- sync with 6.x-1.x-dev.


page.tpl.php
<?php
global $base_url;

global $theme_path;

// $Id: page.tpl.php,v 1.2.2.15 2009/04/25 06:19:22 hswong3i Exp $
?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php print $language->language ?>" lang="<?php print $language->language ?>" dir="<?php print $language->dir ?>">

<head profile="http://gmpg.org/xfn/11">
  <title><?php print $head_title ?></title>
  <?php print $head ?>
  <?php print $styles ?>
  <?php print $scripts ?>
  <!--[if lt IE 7]>
    <?php print phptemplate_get_ie_styles(); ?>
  <![endif]-->
  <script type="text/javascript"><?php /* Needed to avoid Flash of Unstyle Content in IE */ ?> </script>
</head>

<body class="<?php print $body_classes ?>">
<?php if ($header): ?><div id="header-region" class="clear-block"><?php print $header ?></div><?php endif; ?>
<div id="wrapper"><!-- begin wrapper -->
<div id="container" class="clear-block"><!-- begin container -->
  <div class="shadow1">
    <div class="shadow2">
      <div class="shadow3">
	<div id="header"><!-- begin header -->
  		<div id="logo"><a href="http://islandora.coalliance.org" title="Colorado Alliance"></a>
		<a href="http://islandora.coalliance.org" title="Coalliance"><img src="<?php echo $base_url.'/'.$theme_path.'/images/libraries_trans.gif';?>"/></a>
	  <?php if ($logo): ?><a href="<?php print $front_page ?>" title="<?php print $site_name ?>"><img src="<?php print $logo ?>" alt="<?php print $site_name ?>" /></a><?php endif; ?>
	</div>  
	<div id="slogan-floater"><!-- begin slogan-floater -->
	    <?php if ($site_name): ?><h1 class='site-name'><a href="<?php print $front_page ?>" title="<?php print $site_name ?>"><?php print $site_name ?></a></h1><?php endif; ?>
	    <?php if ($site_slogan): ?><div class='site-slogan'><?php print $site_slogan ?></div><?php endif; ?>
	    
	    <div id="utilityLinks"><a href="http://islandora.coalliance.org/">Colorado Alliance</A></div>
	    <div id="search">
                    <form id="frmSearch" action="http://umanitoba.ca/search/newsearch.php" method="get" name="frmSearch">
   <div id="googleSearch" >
        <input id="kw" type="text" name="q" size="26" maxlength="256" value="" class="srchTxt">
    </div>
   
    <div id="googleButton">
        <input value="Search" alt="Search" name="btnG" type="image" class="srchBtn" src="http://umanitoba.ca/media/search.gif">
        <input value="date:D:L:d1" name="sort" type="hidden">
        <input value="xml_no_dtd" name="output" type="hidden">
        <input value="default_collection" name="site" type="hidden">
        <input value="UTF-8" name="ie" type="hidden">
        <input value="UTF-8" name="oe" type="hidden">
        <input value="default_frontend" name="client" type="hidden">
        <input value="default_frontend" name="proxystylesheet" type="hidden">
        <!-- Optional Parameters -->
        <input type='hidden' name='as_dt' value='i'>
        <input type='hidden' name='as_sitesearch' value='http://umanitoba.ca/'>
    </div>
    <div id ="search_parameters" >
<span class="radio1"><input name="search_type" id="searchTypeBtn" value="local" checked="checked" type="radio" style="float:left;"></span>    <span style="float:left;display:inline;" id="text1">Coalliance</span>
<span class="radio2"><input name="search_type" id="people" value="people" type="radio" style="float:left;"></span><span style="float:left;" id="text2">People</span>
<span class="radio3"> <input name="search_type" id="experts" value="experts" type="radio" style="float:left;"></span><span style="float:left;" id="text3">Research Experts</span>
</div>
    
                                    
</form>
<!-- End Google Search -->                
                </div><!--end search-->    
	    
	  </div><!-- end slogan-floater -->
	</div><!-- end header -->
      </div><!-- end shadow3 -->
    </div><!-- end shadow2 -->
  </div><!-- end shadow1 -->
  <div class="shadow1">
    <div class="shadow2">
      <div class="shadow3">
	<div id="primary-nav">
	  <div class="searchbar">
	    <div class='label'>Keyword Search:</div>	    
	    <?php print drupal_get_form('islandora_solr_simple_search_form');?>
	  </div><!-- end searchbar -->
	  <?php if (isset($primary_links)) : ?><!-- begin primary_links -->
	    <?php print theme('links', $primary_links, array('class' => 'primary-links')) ?>
	  <?php endif; ?><!-- end primary_links -->
	</div><!-- end primary-nav-->
      </div><!-- end shadow3 -->
    </div><!-- end shadow2 -->
  </div><!-- end shadow1 -->	
  <?php if ($breadcrumb): print '<div id="breadcrumb">'.$breadcrumb.'</div>'; endif; ?>
  <?php if ($mission): print '<div id="mission">'. phptemplate_mission() .'</div>'; endif; ?>
  <?php if ($left) { ?>
    <div id="sidebar-left" class="sidebar"><!-- begin sidebar-left -->
      <?php if ($search_box): ?><div class="block block-theme"><?php print $search_box ?></div><?php endif; ?>
      <?php print $left ?>
    </div><!-- end sidebar-left -->
  <?php } ?>
  <div id="center"><div id="squeeze"><!-- begin center -->
	<?php if ($title && !$is_front): print '<h2 class="title'. ($tabs ? ' with-tabs' : '') .'">'. $title .'</h2>'; endif; ?>
		<div class="clear-block">
			<?php print $content ?>
		</div>
	  <br clear="all"/>
    <?php if ($tabs): print '<div class="tabs">'. $tabs .'</div>'; endif; ?>
    <?php if ($show_messages && $messages): print $messages; endif; ?>
    <?php print $help ?>
    <?php print $feed_icons ?>
  </div></div><!-- end center -->
  <?php if ($right) { ?>
    <div id="sidebar-right" class="sidebar"><!-- begin sidebar-right -->
      <?php if (!$left && $search_box): ?><div class="block block-theme"><?php print $search_box ?></div><?php endif; ?>
      <?php print $right ?>
    </div><!-- end sidebar-right -->
  <?php } ?>
  <div id="footer"><!-- start footer -->
    <?php print $footer_message ?>
    <?php print $footer ?>
    <span class="copyright">&copy; <?php print date('Y');?> University of Manitoba </span>
  </div><!-- end footer -->
</div><!-- end wrapper -->
</div><!-- end container -->
<?php print $closure ?>
</body>
</html>
